suricata/update/main.py

```diff
@@ -985,7 +992,7 @@ def load_sources(suricata_version):
     # If --etopen is on the command line, make sure its added. Or if
     # there are no URLs, default to ET/Open.
     if config.get("etopen") or not urls:
-        if not urls:
+        if not config.args().offline and not urls:
```

Comment:
I believe this would be taken care of in the fetch function itself? So, this check should not be required.

Reply:
If I remove this check the output would be something like this:
```
25/6/2019 -- 13:05:32 - <Info> -- No sources configured, will use Emerging Threats Open
25/6/2019 -- 13:05:32 - <Info> -- Proceeding offline, will use latest cached version.
```
IMHO, we are using offline mode there should be no need of logging this info `No sources configured, will use Emerging Threats Open` as there is no requirement of using etopen. This was the reason adding the check here. What are your thoughts on it?

Reply:
yes this makes sense. Thanks for clarifying.